content/test/integration/syncronization/error-handling.spec.ts

```diff
@@ -50,11 +46,11 @@ loadTestEnvironment()('End 2 end - Error handling', (testEnv) => {
     await runTest(
       FailureReason.DEPLOYMENT_ERROR,
       (_) => {
-        accessChecker.startReturningErrors()
+        // accessChecker.startReturningErrors()
```

Comment:
Please add a todo here, not sure if this is correct

Reply:
there is a todo for these tests (actually being skipped), i'll removing this line since there is no `accessChecker` any more